tests/transformers/test_transformers_peft_model.py

```diff
@@ -0,0 +1,95 @@
+import importlib
+
+import pytest
+import transformers
+from packaging.version import Version
+
+import mlflow
+from mlflow.models import Model
+from mlflow.transformers.peft import get_peft_base_model, is_peft_model
+
+from tests.transformers.test_transformers_model_export import (
+    HF_COMMIT_HASH_PATTERN,
+    model_path,  # noqa: F401
+)
+
+peft_not_installed_cond = {
```

Comment:
```suggestion
pytestmark = pytest.mark.skipif(...)
```

we can use `pytestmark` to apply marks at the module level.

Reply:
ooh didn't know that feature, thanks!!